modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonUplinkClientCodegen.java

```diff
@@ -0,0 +1,1334 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
@cognifloyd As per the discussion in https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301, there's no need for line 3. Please remove it.

Reply:
This is a copy of the PythonClientCodegen.java where I went through and deleted a big chunk and then added the uplink/marshmallow bits. I was focused on the code and so negelected to evaluate any of the copyright notices. Do I need to add an Apache license / copyright notice in all Java files?

Reply:
Yes, please add the Apache license/copyright to those new Java files

Reply:
I guess there were only 2 files that had/needed the copyright notice (now updated). Do any other files need the copyright fixup?